operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/informer/InformerEventSource.java

```diff
@@ -131,12 +132,12 @@ public void onDelete(R resource, boolean b) {
     }
     primaryToSecondaryIndex.onDelete(resource);
     super.onDelete(resource, b);
-    if (acceptedByDeleteFilters(resource, b)) {
+    if (eventAcceptedByDeleteFilters(resource, b)) {
       propagateEvent(resource);
     }
   }
 
-  private synchronized void onAddOrUpdate(Operation operation, R newObject, R oldObject,
+  private synchronized void onAddOrUpdate(ResourceAction operation, R newObject, R oldObject,
```

Comment:
The separate Operation was intentional to exclude delete. I don't think this needs to be changed, it is more explicit that way.

Reply:
That is something completely different from filters.